### PR TITLE
Rename branch to main

### PR DESCRIPTION
**Pull request by mariechatfield**

### CODE REVIEW

File: docs/index.html [hunk 1]
```diff
@@ -1,4 +1,4 @@
-<!doctype html>
```
Comment (katz12) [inline docs/index.html:1]: you also renamed this file?

Comment (mariechatfield) [reply docs/index.html:1]: yes! this build is currently broken—GitHub pages now expects the main file to be called `index.html`

Comment (katz12) [reply docs/index.html:1]: cool ty for fixing!

**Review by katz12 — APPROVED** [review]

**Comment by mariechatfield** [comment]
GitHub pages is live again, but the main build is broken because of an outdated Python library; will leave for the next person who works on this repo to fix because it's out of scope for the branch migration.

**Pull request merged**